FFCDemoPaymentService.Tests/MessagingTests/MessageServiceTests.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.ComponentModel;
+using System.IO;
+using NUnit.Framework;
+using Moq;
+using FFCDemoPaymentService;
+using FFCDemoPaymentService.Messaging;
+using Microsoft.Extensions.DependencyInjection;
+using System.Threading;
+using System.Threading.Tasks;
+
+
+namespace FFCDemoPaymentService.UnitTests.MessagingTests
+{
+
+    public class MessageServiceTests
+    {        
+        MessageConfig messageConfig;
+        MessageService messageService;
+
+        [SetUp]
+        public void Setup()
+        {
+            messageConfig = new MessageConfig();
+            messageConfig.Host = "123";
```

Comment:
Don't think there's a need to set any of these values for the tests that currently exist.

Reply:
I've removed the message service test class as its currently not actually running any tests. This can be re-created once we have switched to SQS.